brief introduction to pseudocode (#615)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

Appraisal Policy
are used with the meanings defined in {{sec-glossary}}.

## Pseudocode Notation {#sec-pseudocode-notation}

> The algorithms in this section are expressed in an informal pseudocode that is intended to be read, not executed.

Reviewer: >informal pseudocode

If the pseudocode is meant to illustrate normative text or prose, then informal is fine. If the pseudocode is meant to be normative, then informal is not fine.

> The algorithms in this section are expressed in an informal pseudocode that is intended to be read, not executed.

Reviewer 2: ```suggestion
The algorithms in pseudocode are meant for readability and not for execution.
```

> The following basic and somewhat casually defined notation is used throughout.

Reviewer 2: ```suggestion
The following pseudocode notation is used throughout this document.
```

### Comments

A comment begins with "//" and extends to the end of the line.

~~~ pseudocode
// this is a comment
~~~

### Types

> All type names in this pseudocode are taken directly from the CDDL rules defined in this document.

Reviewer 2: ```suggestion
Pseudocode Type names are taken directly from the CDDL {{-cddl}}.
```

> All type names in this pseudocode are taken directly from the CDDL rules defined in this document.

Reviewer 2: Reference to that section CDDL Types in RFC 8610 and Appendix for full CDDL in this document.
